### ISSUE TITLE
The version in pip does not work

### ISSUE THREAD

**Issue by pavelk2** [maethor/avatar-generator#2]
```
17:43:24 web.1  | Traceback (most recent call last):
17:43:24 web.1  |   File "/opt/avatar/lib/python2.7/site-packages/gunicorn/workers/sync.py", line 130, in handle
17:43:24 web.1  |     self.handle_request(listener, req, client, addr)
17:43:24 web.1  |   File "/opt/avatar/lib/python2.7/site-packages/gunicorn/workers/sync.py", line 171, in handle_request
17:43:24 web.1  |     respiter = self.wsgi(environ, resp.start_response)
17:43:24 web.1  |   File "/opt/avatar/lib/python2.7/site-packages/flask/app.py", line 1836, in __call__
17:43:24 web.1  |     return self.wsgi_app(environ, start_response)
17:43:24 web.1  |   File "/opt/avatar/lib/python2.7/site-packages/flask/app.py", line 1820, in wsgi_app
17:43:24 web.1  |     response = self.make_response(self.handle_exception(e))
17:43:24 web.1  |   File "/opt/avatar/lib/python2.7/site-packages/flask/app.py", line 1403, in handle_exception
17:43:24 web.1  |     reraise(exc_type, exc_value, tb)
17:43:24 web.1  |   File "/opt/avatar/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
17:43:24 web.1  |     response = self.full_dispatch_request()
17:43:24 web.1  |   File "/opt/avatar/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
17:43:24 web.1  |     rv = self.handle_user_exception(e)
17:43:24 web.1  |   File "/opt/avatar/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
17:43:24 web.1  |     reraise(exc_type, exc_value, tb)
17:43:24 web.1  |   File "/opt/avatar/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
17:43:24 web.1  |     rv = self.dispatch_request()
17:43:24 web.1  |   File "/opt/avatar/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
17:43:24 web.1  |     return self.view_functions[rule.endpoint](**req.view_args)
17:43:24 web.1  |   File "/Users/pavelk/Code/Learning/avatar-flask-heroku/hello.py", line 13, in photo
17:43:24 web.1  |     avatar = Avatar.generate(128, "example@sysnove.fr", "PNG")
17:43:24 web.1  | TypeError: generate() takes exactly 3 arguments (4 given)
```

when I use the module via 

```
git+https://github.com/maethor/avatar-generator.git
```

it works well

**Comment by pavelk2** [comment]
BTW - i have deployed the version here:
https://avatarius.herokuapp.com/get?name=Pavel&size=128
and the repository with Heroku deploy is here
https://github.com/Codesignio/avatar-flask-heroku
I put a reference on your repository as it is 100% your work.

**Comment by maethor** [comment]
Thanks! I'll look at it tomorrow and update the pip package.

**Comment by sidmitra** [comment]
@maethor I believe the pypi package is still the old version. I go the same error.

**Comment by maethor** [comment]
I just published v0.2 to fix this.
